### PR TITLE
LPS-102933

### PR DESCRIPTION
**Pull request by smailbiala**

### CODE REVIEW

**Comment by ilyasshss** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/MyTests/MyTests.macro [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+		Form.save();
+
+		Form.gotoAddField(
+			fieldPositionNumber = 1,
+			fieldType = "Numeric");
+
+		Form.editFieldLabelMetal(fieldValue = ${ageLabelEnglish});
+
+		Form.save();
```
Comment (ilyasshss) [inline portal-web/test/functional/com/liferay/portalweb/macros/MyTests/MyTests.macro:28]: In this macro, you don't only create form, you also edit and save, so the name of the macro doesnt match what it does

File: portal-web/test/functional/com/liferay/portalweb/macros/MyTests/MyTests.macro [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+		Click(
+			key_name = "Age",
+			locator1 = "FormFields#FIELD_TARGET_LABEL");
+
+		FormFields.editText(
+			fieldName = "Label",
+			fieldValue = ${ageLabelFrench});
+
+		Form.save();
+	}
```
Comment (ilyasshss) [inline portal-web/test/functional/com/liferay/portalweb/macros/MyTests/MyTests.macro:79]: same here, the name of the macro doesnt match what it does

File: portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyFormTest.testcase [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	tearDown {
+		User.logoutAndLoginPG(
+			userLoginEmailAddress = "test@liferay.com",
+			userLoginFullName = "Test Test");
+	}
+
+	@description = "Create a New form with Text feild and Numeric feild"
+	@priority = 5
+	test AddFormWithTextFeildAndNumericFeild {
+		task ("Given Create The Form With The Feilds") {
```
Comment (ilyasshss) [inline portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyFormTest.testcase:23]: minor typo here "Fields"

File: portal-web/test/functional/com/liferay/portalweb/macros/MyTests/MyTests.macro [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+		Form.gotoAddField(
+			fieldPositionNumber = 1,
+			fieldType = "Numeric");
+
+		Form.editFieldLabelMetal(fieldValue = ${ageLabelEnglish});
+
+		Form.save();
+	}
+
+	macro modifyWithHeightAndAplyCustomSize {
```
Comment (ilyasshss) [inline portal-web/test/functional/com/liferay/portalweb/macros/MyTests/MyTests.macro:31]: minor typo "modifyWithHeightAndApplyCustomSize"
this macro modifies both height and width so the name should be something like "modifyHeightAndWidthAndApplyCustomSize"

File: portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyTestFile.testcase [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+	test ValidateConfigurationStringAtControlPanel {
+		task ("Given Open The Application Menu") {
+			ApplicationsMenuHelper.openApplicationsMenu();
+		}
+
+		task ("When Switch To Control Panel") {
+			ApplicationsMenu.gotoPanel(panel = "Control Panel");
+		}
+
+		task ("Then: Validate The Configuration is Displayed") {
```
Comment (ilyasshss) [inline portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyTestFile.testcase:58]: Then should not have colon after it

**Review by ilyasshss — COMMENTED** [review]
Minor changes like typos and renaming some macros

**Comment by smailbiala** [comment]
ci:close

**Pull request closed without merging**